apps/cowswap-frontend/src/common/pure/CurrencyInputPanel/defaultCurrencyInputProps.ts

```diff
@@ -28,6 +28,8 @@ export const defaultCurrencyInputPanelProps: CurrencyInputPanelProps & { priceIm
       amountAfterFeesRaw: CurrencyAmount.fromRawAmount(currency, 20 * 10 ** 18),
       feeAmount: '10',
       feeAmountRaw: CurrencyAmount.fromRawAmount(currency, 10 * 10 ** 18),
+      partnerFeeAmount: '25',
+      partnerFeeAmountRaw: CurrencyAmount.fromRawAmount(currency, 25 * 10 ** 18),
```

Comment:
Why does this default to non-zero amounts?

Reply:
Changed to 0